packages/nextjs/components/scaffold-eth/Contract/DisplayVariables.tsx

```diff
@@ -25,6 +26,12 @@ const DisplayVariable = ({ contractAddress, functionFragment }: TDisplayVariable
     },
   });
 
+  useEffect(() => {
+    (async () => {
+      await refetch();
+    })();
```

Comment:
Since we are not doing anything with the response || catching errors, we can just call `refetch()` with no async/await